src/include/processor/operator/simple/use_database.h

```diff
@@ -5,6 +5,21 @@
 namespace kuzu {
 namespace processor {
 
+struct UseDatabasePrintInfo final : OPPrintInfo{
+    std::string dbName;
+
+    UseDatabasePrintInfo(std::string dbName)
+        : dbName(std::move(dbName)) {}
+    UseDatabasePrintInfo(const UseDatabasePrintInfo& other)
```

Comment:
Since the copy constructor doesn't need to be public, we have changed it to be private. Here is an example of how to make it private. 
https://github.com/kuzudb/kuzu/blob/bfb2480ffacbdf0520ba2846ba293c80f1f7486e/src/include/processor/operator/ddl/alter.h#L21-L29
Move the copy constructor to be private and switch `make_unique` in the copy function to `unique_ptr`.